### PR TITLE
test: install cypress_on_rails and add a simple test as an example

### PR DESCRIPTION
**Pull request by beagleknight**
This setup the e2e test suite for our bulletin board application. It will test our sandbox environment, so we can be sure that both the JS and the Ruby clients work.

I added a simple test as an example, so we can start building something on top of that.

### CODE REVIEW

File: .github/workflows/app-workflow.yml [hunk 1]
```diff
@@ -123,3 +123,71 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           directory: ./decidim-bulletin_board-app/coverage
+  e2e-test:
```
Comment (leio10) [inline .github/workflows/app-workflow.yml:126]: 😍 🔝 ❤️ 🤩 🥇 💎

**Review by leio10 — APPROVED** [review]
This is soooooo good!! 👏👏👏

**Review by orlera — APPROVED** [review]
🧷

**Review by agustibr — APPROVED** [review]
👏👏

**Pull request merged**